commands/info.md

```
@@ -0,0 +1,50 @@
+---
+description: Query Unraid server information and configuration
+argument-hint: [action] [additional-args]
+---
+
+Execute the `unraid_info` MCP tool with action: `$1`
```

markdownlint-cli2 (0.20.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)